p2p/pex/addrbook.go

```diff
@@ -627,7 +629,9 @@ func (a *addrBook) addAddress(addr, src *p2p.NetAddress) error {
 	}
 
 	bucket := a.calcNewBucket(addr, src)
-	a.addToNewBucket(ka, bucket)
+	if err := a.addToNewBucket(ka, bucket); err != nil {
+		return err
+	}
 	return nil
```

Comment:
```suggestion
	return a.addToNewBucket(ka, bucket)
```

This can be shorten to this.

Reply:
Thanks! changed it =)